lib/cachex/services/courier.ex

```diff
@@ -39,8 +39,15 @@ defmodule Cachex.Services.Courier do
   call which will wait until a result can be loaded.
   """
   @spec dispatch(Cachex.t(), any, (-> any)) :: any
-  def dispatch(cache() = cache, key, task) when is_function(task, 0),
-    do: service_call(cache, :courier, {:dispatch, key, task, local_stack()})
+  def dispatch(cache() = cache, key, task) when is_function(task, 0) do
```

Comment:
Same for this, prefer the one-line that was there before with `get_callers()` inlined.